pathfinder/tests/fixtures/network_service.py

```diff
@@ -23,6 +24,17 @@ def forge_fee_signature(private_key: str, fee: float) -> bytes:
     return private_key_ecdsa.sign_recoverable(fee_msg)
 
 
+@pytest.fixture
```

Comment:
use `yield` to return listener from the fixture and stop it manually, like here for example:

https://github.com/raiden-network/raiden-monitoring-service/blob/master/monitoring_service/test/fixtures/server.py#L54

Reply:
Done.